rosdep/oe-opkg.yaml

```diff
@@ -0,0 +1,335 @@
+asio:
+  oe:
+    opkg:
+      packages: [asio@meta-oe]
```

Comment:
All of these rules follow the oe -> opkg -> packages it would be great to make this more compact if the oe key with values attached would default to using opkg as the default provider and assuming that its a list of packages. 

This would remove over 50% of the lines to support this.

Reply:
Agree; let's modify base.yaml and python.yaml as you suggested so it will be just one extra line per key for the openembedded line when it's actually needed.